### PR TITLE
docs: add Globe deployment documentation

### LINKED ISSUE #1192: docs: Add Globe to Deployment section in Docs
**Description**

With the launch of Globe it looks like they have a section in their documentation on Dart Frog deployment, https://docs.globe.dev/frameworks/dart-frog. Let's add this to our deployment docs as well.

**Requirements**

- [ ] We have tested deployment of a sample Dart Frog server/project on Globe
- [ ] The deployment steps are documented in the Dart Frog docs

### PR DESCRIPTION
**Pull request by alestiago**
## Status

**READY**

## Description

Resolves #1192

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [X] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: docs/docs/deploy/globe.md
Comment (wolfenrain) [inline docs/docs/deploy/globe.md:31]: According to the docs this is a preview deployment and not a production deployment

**Review by wolfenrain — DISMISSED** [review]
LGTM, approved but let's see if @invertase has any opinions on it before merging

**Comment by alestiago** [comment]
Drafting to avoid accidentally merging.

**Review by Salakar — APPROVED** [review]

File: docs/docs/deploy/globe.md
Comment (dackers86) [inline docs/docs/deploy/globe.md:39]:
```suggestion
- [`globe deploy` documentation](https://docs.globe.dev/cli/commands/deploy)
```

**Review by dackers86 — APPROVED** [review]

**Review by Ehesp — APPROVED** [review]
Lgtm

**Review by wolfenrain — APPROVED** [review]

**Pull request merged**
